package.json

```diff
@@ -6,28 +6,28 @@
   "dependencies": {
     "express": "4.18.2",
```

Comment:
should we just upgrade to 5 ?

Reply:
I don't think so, the version tagged as `latest`on npm is [4.21.1](https://www.npmjs.com/package/express/v/4.21.1)